src/applications/caregivers/tests/e2e/utils/helpers.js

```diff
@@ -176,4 +182,15 @@ export const pageHooks = {
       }
     });
   },
+  confirmation: ({ afterHook }) => {
+    afterHook(() => {
+      cy.get('va-link')
+        .contains('Download your completed application (PDF)')
```

Comment:
I'm not in love with validating by content. We do have the string in the locale file. Maybe we at least include that string here instead of hardcoding text.

Reply:
Is there a good way to get the `va-link` another way? Using the content does make more sense, with the caveat that with how the link is generated we'll have to parse it to add the `(PDF)` on. So something like this maybe? 

```
    afterHook(() => {
      const downloadLinkText = `${content['button-download']} (PDF)`;

      cy.get('va-link')
        .contains(downloadLinkText)
        .click();

      cy.wait('@downloadPdf').then(() => {
        cy.get('va-link').contains(downloadLinkText);
      });
    });
```

Reply:
I think you can just do contains with the string. I'm pretty sure it doesn't look for an exact match of what you provide, just that what you provide is there. You could also just add a class or `data-testid` attribute to the `va-link` component and query by that.